ci/ray_ci/tester.py

```diff
@@ -445,6 +450,20 @@ def _get_human_specified_tests() -> Set[str]:
     return tests
 
 
+def _get_new_tests(prefix: str, container: TesterContainer) -> Set[str]:
+    """
+    Get all local test targets that are not in database
+    """
+    local_test_targets = set(
+        container.run_script_with_output(['bazel query "tests(//...)"'])
+        .strip()
+        .split(os.linesep)
```

Comment:
you might just call `split()` ?

Reply:
i remember split() breaks in windows without `os.linestep`